bitnami/geode/templates/locator/statefulset.yaml

```diff
@@ -0,0 +1,387 @@
+                  key: keystore-password
+            {{- end }}
+            {{- if not (empty .Values.auth.tls.truststorePassword) }}
+            - name: GEODE_SECURITY_TLS_TRUSTSTORE_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "geode.secretName" . }}
+                  key: truststore-password
+            {{- end }}
+            {{- end }}
+            {{- if .Values.locator.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.locator.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          {{- if or .Values.locator.extraEnvVarsCM .Values.locator.extraEnvVarsSecret }}
+          envFrom:
+            {{- if .Values.locator.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.locator.extraEnvVarsCM "context" $) }}
+            {{- end }}
+            {{- if .Values.locator.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.locator.extraEnvVarsSecret "context" $) }}
+            {{- end }}
+          {{- end }}
+          {{- if .Values.locator.resources }}
+          resources: {{- toYaml .Values.locator.resources | nindent 12 }}
+          {{- end }}
+          ports:
+            - name: locator
+              containerPort: {{ .Values.locator.containerPorts.locator }}
+            - name: http
+              containerPort: {{ .Values.locator.containerPorts.http }}
+            - name: rmi
+              containerPort: {{ .Values.locator.containerPorts.rmi }}
+            {{- if .Values.metrics.enabled }}
+            - name: int-metrics
+              containerPort: {{ .Values.locator.containerPorts.metrics }}
+            {{- end }}
+          {{- if not .Values.diagnosticMode.enabled }}
+          {{- if .Values.locator.livenessProbe.enabled }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.locator.livenessProbe "enabled") "context" $) | nindent 12 }}
+            exec:
+              command:
+                - /bin/bash
+                - -ec
+                - |
+                  gfsh status locator --dir /bitnami/geode/data >/dev/null 2>&1 || exit 1
+          {{- else if .Values.locator.customLivenessProbe }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.locator.customLivenessProbe "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.locator.readinessProbe.enabled }}
+          readinessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.locator.readinessProbe "enabled") "context" $) | nindent 12 }}
+            exec:
+              command:
+                - /bin/bash
+                - -ec
+                - |
+                  gfsh status locator --dir /bitnami/geode/data >/dev/null 2>&1 || exit 1
```

Comment:
For usability purposes, have you considered checking if there is any way to configure Geode to locate the proper data dir without the need for specifying `--dir`? I.e. by changing a configuration or adding a symlink to the `../data` dir basing on where the `gfsh` binary is located.

Reply:
It automatically looks for a Geode installation in the working directory. That's why I need the flag.

Reply:
So no way to workaround it? I.e. via config files or something else.

Reply:
None that I'm aware of